plugin/db/spanner/spanner.go

```diff
@@ -110,6 +142,14 @@ func (*Driver) Query(_ context.Context, _ string, _ *db.QueryContext) ([]interfa
 	panic("not implemented")
 }
 
+func (d *Driver) composeDSN() string {
```

Comment:
Any reason we need to have two methods doing similar things?

Reply:
removed